dockerhub-readme.md

```diff
@@ -0,0 +1,141 @@
+#  RAPIDS - Open GPU Data Science
+
+## What is RAPIDS?
+
+Visit [rapids.ai](https://rapids.ai) for more information.
+
+The RAPIDS suite of software libraries gives you the freedom to execute end-to-end data science and analytics pipelines entirely on GPUs. It relies on NVIDIA® CUDA® primitives for low-level compute optimization, but exposes GPU parallelism and high-bandwidth memory speed through user-friendly Python interfaces.
+
+**NOTE:** Review our [system requirements](https://docs.rapids.ai/install#system-req) to ensure you have a compatible system!
+
+
+### Current Version - RAPIDS v23.08
+
+Libraries included in the images:
+- `cuDF`
+- `cuML`
+- `cuGraph`
+- `RMM`
+- `RAFT`
+- `cuSpatial`
+- `cuSignal`
+- `cuxfilter`
+- `cuCIM`
+- `xgboost`
+- `dask-sql`
+
+
+### Image Types
+
+The RAPIDS images are based on [`nvidia/cuda`](https://hub.docker.com/r/nvidia/cuda) and [`rapidsai/mambaforge-cuda`](https://hub.docker.com/r/rapidsai/mambaforge-cuda). The RAPIDS images provide `amd64` & `arm64` architectures where supported.
+
+
+There are two types:
+- `rapidsai/base` - contains a RAPIDS environment ready for use.
+  - **TIP: Use this image if you want to use RAPIDS as a part of your pipeline.**
+- `rapidsai/notebooks` - extends the `rapidsai/base` image by adding a [`jupyterlab` server](https://jupyter.org/), example notebooks, and dependencies.
+  - **TIP: Use this image if you want to explore RAPIDS through notebooks and examples.**
+
+### Image Tag Naming Scheme
+
+The tag naming scheme for RAPIDS images incorporates key platform details into the tag as shown below:
+```
+23.08-cuda11.8-py3.10
+ ^        ^      ^
+ |        |      Python version
+ |        |
+ |        CUDA version
+ |
+ RAPIDS version
+```
+
+**Note: Nightly builds of the images have the RAPIDS version appended with an `a` (ie `23.08a-cuda11.8-py3.10`)**
+
+## Usage
+
+The `rapidsai/base` image starts with an [`ipython` shell](https://ipython.org/) by default.
+
+The `rapidsai/notebooks` image starts with the [JupyterLab notebook server](https://jupyter.org/) by default.
+
+### Container Ports
+
+`rapidsai/notebooks` exposes port `8888` for the [JupyterLab notebook server](https://jupyter.org/).
+
+### Environment Variables
+
+The following environment variables can be passed to the `docker run` commands:
+
+- `EXTRA_CONDA_PACKAGES` - used to install additional `conda` packages in the container. Use a space separated list of values
+- `CONDA_TIMEOUT` - how long (in seconds) the `conda` command should wait before exiting
+- `EXTRA_PIP_PACKAGES` - used to install additional `pip` packages in the container. Use a space separated list of values
+- `PIP_TIMEOUT` - how long (in seconds) the `pip` command should wait before exiting
+
+Example:
+
+```sh
+$ docker run \
+    --rm \
+    -it \
+    --gpus all \
```

Comment:
we've added the `--pull always` command to the selector, we could add it here too if we want so then if a users copies this without pulling it will get them the image too.